galasa-ui/messages/en.json

```diff
@@ -245,6 +245,7 @@
       "default": "Showing test runs submitted in the last 24 hours",
       "range": "Showing test runs submitted between {from} and {to}"
     },
+    "noColumnsSelected": "No columns selected – nothing to display. Please select one or more columns.",
```

Comment:
This is good but could be clarified to mention the table design tab, maybe something like:

"All of the columns have been hidden in the table design tab, so no result details will be visible. Please adjust the table design to show the columns that you would like to see."

Feel free to come up with a better message!